src/liquid/components/search_results.html

```diff
@@ -11,7 +11,10 @@
           {% if searchResult.type == 'local' %}
             <span class="material-symbols-outlined">cloud_off</span>
           {% endif %}
-          {{searchResult.name}}
+          {{searchResult.name}} 
+          {% if searchResult.lineage.length > 0 %}
```

Comment:
This is what I'm seeing for Kenya... Is this what you intended?
![image](https://github.com/medic/cht-user-management/assets/9014751/06d82edb-5698-492f-b43b-74db28266394)

Reply:
@kennsippell no, that was not my intention. Let me take a look at this behaviour